bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx95-de.html

```diff
@@ -0,0 +1,87 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% from "macros-protocol.html" import card with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}Was ist neu in Firefox? – Mehr Privatsphäre, mehr Schutz.{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}Was ist neu in Firefox? – Mehr Privatsphäre, mehr Schutz.{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_95_de') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+      <div class="mzp-c-logo mzp-t-logo-md mzp-t-product-pocket">Pocket</div>
+      <h1 class="wnp-main-title">Best of Pocket 2021 ist da!</h1>
+      <p class="wnp-main-tagline">
+        In Pocket findest du die interessantesten und gehaltvollsten Inhalte aus dem Internet. Lass dich inspirieren und tauche ein in die meistgelesenen Geschichten des Jahres.
+      </p>
+
+      <div class="wnp-main-cta">
+        {{ pocket_fxa_button(entrypoint=entrypoint, button_text='Jetzt anmelden und lesen', optional_parameters={'s': entrypoint, 'utm_campaign': utm_campaign, 'utm_content': 'signup-fxa-primary'}, optional_attributes={'data-cta-text': 'Sign up and read now', 'data-cta-type': 'button', 'data-cta-position': 'primary'}) }}
+      </div>
+    </div>
+  </section>
+
+  <section class="wnp-content-extra">
+    <div class="mzp-l-content mzp-t-content-lg mzp-l-columns mzp-t-columns-three">
+      <div class="mzp-c-picto">
```

Comment:
Curious: why not use macros here?

Reply:
Only because I'm using inline SVG assets here (for better WNP performance), rather than adding additional HTTP requests for the images.

Reply:
<img width="335" alt="Screen Shot 2021-11-26 at 11 13 18 AM" src="https://user-images.githubusercontent.com/19650432/143574354-50cd1291-6029-4669-a44c-3f06ca16dbb9.png">

since we're inlining svg but there's no valuable information we need to convey, it's probably worth adding `aria-hidden=true` on the svgs to remove them from the accessibility tree